src/types/emr/encounter.ts

```diff
@@ -78,18 +87,20 @@ export const ENCOUNTER_PRIORITY = [
   "use_as_directed",
 ] as const;
 
-export const ENCOUNTER_STATUS = [
-  "planned",
-  "in_progress",
-  "on_hold",
-  "discharged",
-  "completed",
-  "cancelled",
-  "discontinued",
-  "entered_in_error",
-  "unknown",
+export const ENCOUNTER_STATUSES = [
+  { id: "planned", icon: "l-calender" },
+  { id: "in_progress", icon: "l-spinner" },
+  { id: "on_hold" },
+  { id: "discharged", icon: "l-home" },
+  { id: "completed", icon: "l-check" },
+  { id: "cancelled", icon: "l-x" },
+  { id: "discontinued" },
+  { id: "entered_in_error" },
+  { id: "unknown" },
 ] as const;
 
+export const ENCOUNTER_STATUS = ENCOUNTER_STATUSES.map((status) => status.id);
```

Comment:
let's not change this. 

instead we can create a new record called: `ENCOUNTER_STATUS_ICONS` whose keys would be the encounter status and values the icons as react element